### DISCUSSION TITLE
Tailwind standalone can support postcss.config.js

### DISCUSSION THREAD

**Discussion by cvkmohan** [tailwindlabs/tailwindcss#7262] [Ideas]
If the standalone tailwind can support the ```postcss.config.js```, it will be very useful. Specially, if we want to use ```tailwind/nested```, it can provide a very convenient method - without the need to go with the full npm installation method. This is relevant for people coming from Rails and Phoenix frameworks - where it is more common to use the standalone than an npm package.

**Comment by Xeevis** [comment]
Well technically it already does, you can provide custom postcss.config.js with  `--postcss` arg, but I think you then need to also provide custom plugins with NPM which totally defeats purpose of a Standalone CLI. 

I'm not a fan of the whole integrated build environment. What I do instead is to use Node's younger brother Deno which is just a single executable to separate the concern between tools, I let the PostCSS deal with imports & nesting and the resulting css is processed by tailwindcss CLI which just deals with `@apply`, `@layer` and JIT contents. I wish Tailwindcss just dropped the whole PostCSS/Node dependency and became fast standalone rust CLI. Any preprocessing and postprocessing can be done by other tools of own choice.

**Reply by cvkmohan** [reply]
Hmm... Yeah - it would be great if Tailwind just drops that PostCSS dependency.  Atleast if they switch to https://github.com/parcel-bundler/parcel-css - that might ease a lot of things for both them and us. Take the parcel-css rust binary and use it to process the entire css - and then produce a CLI - that might be easy on everybody and still very fast.
